core/src/main/scala/nl/vroste/zio/kinesis/client/zionative/Consumer.scala

```diff
@@ -234,12 +234,12 @@ object Consumer {
       streamDescription: StreamDescription.ReadOnly
     ): ZIO[Scope with Kinesis, Throwable, Fetcher] =
       fetchMode match {
-        case c: Polling        => PollingFetcher.make(streamDescription.streamName, c, emitDiagnostic)
+        case c: Polling        => PollingFetcher.make(StreamIdentifier.fromARN(streamDescription.streamARN), c, emitDiagnostic)
```

Comment:
Let's adjust PollingFetcher.make and EnhancedFanoutFetcher.make to accept only a StreamIdentifierByArn, currently they're a bit inconsistent.